Sources/tart/MACAddressResolver/ARPCache.swift

```diff
@@ -52,7 +52,12 @@ struct ARPCache {
     process.standardInput = FileHandle.nullDevice
 
     try process.run()
-    process.waitUntilExit()
+    while process.isRunning {
+      if Task.isCancelled {
+        process.terminate()
```

Comment:
It seems to me that this line won't be called 99.99% of the time, because `Task.sleep()` runs long enough and will throw a `CancellationError`, thus exiting this loop.

What do you think about addressing the root cause instead? See https://github.com/cirruslabs/tart/pull/916.